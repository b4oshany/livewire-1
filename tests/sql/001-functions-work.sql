\unset ECHO
\i tests/tests.sql

/*******************************************************

Test Suite for LiveWire


*******************************************************/
SELECT lw_initialise('burlap',3450);


SELECT plan(5);



SELECT has_schema(
  'burlap', 
  'Check for existence of burlap after running lw_initialise');

SELECT has_table(
  'burlap', 
  'burlap',
  'Check for the existence of burlap.burlap after running lw_initialise');

SELECT has_table(
  'burlap',
  '__livewire', 
  'Check for the existence of burlap.__livewire after running lw_initialise');

SELECT has_table(
  'burlap',
  '__lines',
  'Check for the existence of burlap.__lines after running lw_initialise');

SELECT has_table(
  'burlap',
  '__nodes',
  'Check for the existence of burlap.__nodes after running lw_initialise');

SELECT * FROM finish();

ROLLBACK;
