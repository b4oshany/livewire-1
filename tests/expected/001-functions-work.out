\unset ECHO
1..5
ok 1 - Check for existence of burlap after running lw_initialise
ok 2 - Check for the existence of burlap.burlap after running lw_initialise
ok 3 - Check for the existence of burlap.__livewire after running lw_initialise
ok 4 - Check for the existence of burlap.__lines after running lw_initialise
ok 5 - Check for the existence of burlap.__nodes after running lw_initialise
